utils/CMakeLists.txt

```diff
@@ -6,9 +6,9 @@ include_directories(${NIFTI_SOURCE_DIR}/niftilib ${NIFTI_SOURCE_DIR}/znzlib ${NI
 add_executable(${PACKAGE_PREFIX}nifti_stats nifti_stats.c)
 add_executable(${PACKAGE_PREFIX}nifti_tool nifti_tool.c)
 add_executable(${PACKAGE_PREFIX}nifti1_test nifti1_test.c)
-target_link_libraries( nifti_stats ${PACKAGE_PREFIX}niftiio ${PACKAGE_PREFIX}nifticdf)
-target_link_libraries( nifti_tool  ${PACKAGE_PREFIX}niftiio)
-target_link_libraries( nifti1_test ${PACKAGE_PREFIX}niftiio)
+target_link_libraries( nifti_stats PUBLIC ${PACKAGE_PREFIX}niftiio ${PACKAGE_PREFIX}nifticdf)
+target_link_libraries( nifti_tool  PUBLIC ${PACKAGE_PREFIX}niftiio)
```

Comment:
nifti_stats is not worth building. nifti1_test should be moved to tests and  the nifti_tool should utility should be built from the src in nifti2 (which is compatible with both all nifti files). Can you confirm this @afni-rickr ?